python/paddle/fluid/layers/detection.py

```diff
@@ -275,13 +275,13 @@ class number, M is number of bounding boxes. For each category
     Examples:
         .. code-block:: python
 
-            pb = layers.data(name='prior_box', shape=[10, 4],
+            pb = fluid.layers.data(name='prior_box', shape=[10, 4],
```

Comment:
Please add `import paddle.fluid as fluid`

Reply:
Done in all APIs. Thanks!